.github/workflows/ci.yml

```diff
@@ -137,10 +137,10 @@ jobs:
           path: samples/artifacts/builds/Android
 
       - name: Build Standalone Player with IL2CPP
-        run: dotnet msbuild /t:UnityBuildPlayerIL2CPP /p:Configuration=Release
+        run: dotnet msbuild /t:UnityBuildStandalonePlayerIL2CPP /p:Configuration=Release
```

Comment:
Now that we have "IL2CPP Players", naming it so it's easier to differentiate between Standalone, iOS, Android..